python/common/business-registry-dissolution/tests/__init__.py

```diff
@@ -0,0 +1,173 @@
+# Copyright © 2025 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module holds general utility functions and helpers for the main package."""
+from datedelta import datedelta
+import base64
+import uuid
+from business_common.utils.datetime import datetime
+from datetime import datetime as _datetime, timezone
+from contextlib import contextmanager
+from freezegun import freeze_time
+
+from business_model.models import (
+    Batch,
+    BatchProcessing,
+    Business,
+    db,
+    Filing,
+)
+
+from business_model.models.db import VersioningProxy
+
+
+EPOCH_DATETIME = datetime.from_date(_datetime(1970, 1, 1, tzinfo=timezone.utc))
+FROZEN_DATETIME = _datetime(2001, 8, 5, 7, 7, 58, 272362, tzinfo=timezone.utc)
+
+def factory_batch(batch_type=Batch.BatchType.INVOLUNTARY_DISSOLUTION,
+                  status=Batch.BatchStatus.HOLD,
+                  size=3,
+                  notes=''):
+    batch = Batch(
+        batch_type=batch_type,
+        status=status,
+        size=size,
+        notes=notes
+    )
+    batch.save()
+    return batch
+
+
+def factory_batch_processing(batch_id,
+                             business_id,
+                             identifier,
+                             step=BatchProcessing.BatchProcessingStep.WARNING_LEVEL_1,
+                             status=BatchProcessing.BatchProcessingStatus.PROCESSING,
+                             trigger_date=datetime.utcnow() + datedelta(days=42),
+                             notes=''
+                             ):
+    batch_processing = BatchProcessing(
+        batch_id=batch_id,
+        business_id=business_id,
+        business_identifier=identifier,
+        step=step,
+        status=status,
+        trigger_date=trigger_date,
+        notes=notes
+    )
+    batch_processing.save()
+    return batch_processing
+
+def factory_business(identifier,
```

Comment:
these factory business / filing methods are probably a good candidate to move into the common model. Not something to change now I think, but to keep in mind for later